src/views/shortcodes/my-attendance-list.php

```diff
@@ -6,22 +6,42 @@
  *
  *     [your-theme]/tribe-events/shortcodes/my-attendance-list.php
  *
- * @version 4.8.2
+ * @since   4.8.2
+ * @since   TBD Removed target="_blank" from links, added direct link to each post's "My Tickets" view,
+ *          rename $event_id variable.
+ *
+ * @version TBD
  *
  * @var array $event_ids
  */
+
+$view = Tribe__Tickets__Tickets_View::instance();
 ?>
 
 <ul class="tribe-tickets my-attendance-list">
-	<?php foreach ( $event_ids as $id ): ?>
-		<?php $start_date = tribe_get_start_date( $id ); ?>
-		<li class="event-<?php echo esc_attr( $id ) ?>">
-			<a href="<?php echo esc_url( get_permalink( $id ) ); ?>" target="_blank">
-				<?php echo get_the_title( $id ); ?>
+	<?php
+	foreach ( $event_ids as $event_id ) :
+		$is_event               = function_exists( 'tribe_is_event' ) ? tribe_is_event( $event_id ) : false;
+		$direct_link_my_tickets = $view->get_tickets_page_url( $event_id, $is_event );
+		?>
+		<?php $start_date = tribe_get_start_date( $event_id ); ?>
+		<li class="event-<?php echo esc_attr( $event_id ); ?>">
+			<a href="<?php echo esc_url( get_permalink( $event_id ) ); ?>" class="event-post-link">
+				<?php echo get_the_title( $event_id ); ?>
```

Comment:
:no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_the_title'.

<sub>hash: 83dfefc66d1105b831e5f3605ea4d0cc</sub>